src/operator/tensor/broadcast_reduce_op_value.cc

```diff
@@ -15,6 +15,13 @@ DMLC_REGISTER_PARAMETER(BroadcastToParam);
 inline std::string get_reduce_axes_description(const std::string& op_name, int line) {
   std::string doc = R"code(Computes the __op__ of array elements over given axes.
 
+The parameter `axis` specifies axis or axes along which to perform the reduction.
```

Comment:
Mu mentioned 

> The reason there is a \n\n is because we want the following format:

> The axis ...

> If axis is int, ...
> If axis is a tuple of xxx
> You can either use R"code( xxx )code" here to reserve this format, or move the if items to > the description of the function.

The `\n` won't work in the parameter description so I moved this part to the function description. Would you prefer to keep this in the parameter description?